src/php/Analyzer/TupleSymbol/IfSymbol.php

```diff
@@ -11,34 +11,51 @@
 use Phel\Lang\Tuple;
 use Phel\NodeEnvironment;
 
+/**
+ * (if condition truthy).
+ * (if condition truthy falsy).
+ */
 final class IfSymbol implements TupleSymbolAnalyzer
 {
     use WithAnalyzer;
 
     public function analyze(Tuple $tuple, NodeEnvironment $env): IfNode
+    {
+        $this->verifyArguments($tuple);
+
+        return new IfNode(
+            $env,
+            $this->condition($tuple, $env),
+            $this->truthy($tuple, $env),
+            $this->falsy($tuple, $env),
+            $tuple->getStartLocation()
+        );
+    }
+
+    private function verifyArguments(Tuple $tuple): void
     {
         $tupleCount = count($tuple);
+
         if ($tupleCount < 3 || $tupleCount > 4) {
             throw AnalyzerException::withLocation("'if requires two or three arguments", $tuple);
         }
+    }
 
-        $testExpr = $this->analyzer->analyze(
-            $tuple[1],
-            $env->withContext(NodeEnvironment::CONTEXT_EXPRESSION)->withDisallowRecurFrame()
-        );
-        $thenExpr = $this->analyzer->analyze($tuple[2], $env);
-        $elseExpr = $this->elseExpr($tuple, $env);
+    private function condition(Tuple $tuple, NodeEnvironment $env): Node
+    {
+        $envWithDisallowRecurFrame = $env
+            ->withContext(NodeEnvironment::CONTEXT_EXPRESSION)
+            ->withDisallowRecurFrame();
 
-        return new IfNode(
-            $env,
-            $testExpr,
-            $thenExpr,
-            $elseExpr,
```

Comment:
I saw that we use in the [control flow docu](https://github.com/jenshaase/phel-lang/tree/master/doc/content/documentation/control-flow.md) this wording: test, then, else.
<img width="1099" alt="Screenshot 2020-10-24 at 15 01 55" src="https://user-images.githubusercontent.com/5256287/97082426-de002400-1609-11eb-9f77-b7cf76d66e7a.png">

But I think `(if condition truthy falsy)` is more expressive. What do you think? If you like the idea, I would create a PR renaming from the docu and related code 
- test -> condition
- then -> truthy
- else -> falsy

Reply:
In other Lisp documentations you will find the wording `test`, `then` and `else`. Therefore, I used it like they did.

Reply:
Ah, then I will modify it in my PR as well to use this same naming